### DISCUSSION TITLE
Training curves for discriminator losses?

### DISCUSSION THREAD

**Discussion by turian** [lucidrains/audiolm-pytorch#178] [Q&A]
I am curious if someone can share what the training curves look like for the discriminator losses?

@djqualia perhaps?

I am wondering if on my particular corpus, they are dropping fast enough.

**Comment by djqualia** [comment]
sorry, i'm not having much time these days to be active here.  here's a screenshot of discriminator losses from 0.26.0 for ~11 days of training.

![discr](https://user-images.githubusercontent.com/5771497/235283565-19411728-cf75-45e1-9d3b-3d6264a8b2c5.jpg)

and here's some of the other losses:

![total](https://user-images.githubusercontent.com/5771497/235283696-36519bf9-8260-489c-af23-e3da6ba6de67.jpg)

i had to smooth 0.999 to see these trends (very noisy)
